core/src/test/scala/integration/kafka/server/IntegrationTestUtils.scala

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-package integration.kafka.server
+package kafka.server
```

Comment:
I feel this package name is intentional. @mumrah WDYY?

Reply:
@mumrah Can you take a look?